src/components/product/ProductCard.component.tsx

```diff
@@ -0,0 +1,22 @@
+import buyIcon from '@/assets/buy.svg';
+import type { Product } from '@/interfaces/Product.ts';
+
+import styles from './product.module.css';
+
+export const ProductCardComponent = ({ product }: { product: Product }) => (
+    <div className={styles.card}>
+        <div>
+            <img className={styles.cardImg} src={product.images[0]} alt={'card'} />
+            <h2 className={styles.cardTitle}>{product.title}</h2>
+        </div>
+        <div className={styles.cardPriceWrapper}>
+            <p className={styles.cardPrice}>
+                {product.price}
+                <span className={styles.cardPricePrefix}>&#8372;</span>
+            </p>
+            <button className={styles.cardButton}>
```

Comment:
![Screenshot 2024-05-08 at 18 40 46](https://github.com/3apy6a/fe-react-2024/assets/115080771/e7e52060-3553-409d-acb4-8913c1a59dce)

you need to remove border when hovering over the button

Reply:
done